fortio_main.go

```diff
@@ -174,6 +174,11 @@ var (
 	mirrorOriginFlag = flag.Bool("multi-mirror-origin", true, "Mirror the request url to the target for multi proxies (-M)")
 	multiSerialFlag  = flag.Bool("multi-serial-mode", false, "Multi server (-M) requests one at a time instead of parallel mode")
 	udpTimeoutFlag   = flag.Duration("udp-timeout", udprunner.UDPTimeOutDefaultValue, "Udp timeout")
+
+	accessLogFileFlag = flag.String("access-log-file", "",
+		"file to log all requests to. Maybe have performance impacts")
+	accessLogFileFormat = flag.String("access-log-format", "",
```

Comment:
shouldn't there be a default value, otherwise there is a dead end (someone sets -access-log-file and it fails)

Reply:
done